modules/virt-creating-linux-bridge-nad-cli.adoc

```diff
@@ -30,25 +30,25 @@ kind: NetworkAttachmentDefinition
 metadata:
   name: bridge-network # <1>
   annotations:
-    k8s.v1.cni.cncf.io/resourceName: bridge.network.kubevirt.io/bridge-interface # <2>
+    k8s.v1.cni.cncf.io/resourceName: bridge.network.kubevirt.io/br1 # <2>
 spec:
   config: |
     {
       "cniVersion": "0.3.1",
       "name": "bridge-network", <3>
       "type": "bridge", <4>
-      "bridge": "bridge-interface", <5>
+      "bridge": "br1", <5>
       "macspoofchk": false, <6>
       "vlan": 100, <7>
       "disableContainerInterface": true,
       "preserveDefaultVlan": false <8>
     }
 ----
 <1> The name for the `NetworkAttachmentDefinition` object.
-<2> Optional: Annotation key-value pair for node selection, where `bridge-interface` must match the name of a bridge configured on some nodes. If you add this annotation to your network attachment definition, your virtual machine instances will only run on the nodes that have the `bridge-interface` bridge connected.
+<2> Optional: Annotation key-value pair for node selection for the bridge configured on some nodes. If you add this annotation to your network attachment definition, your virtual machine instances will only run on the nodes that have the defined bridge connected.
 <3> The name for the configuration. It is recommended to match the configuration name to the `name` value of the network attachment definition.
 <4> The actual name of the Container Network Interface (CNI) plugin that provides the network for this network attachment definition. Do not change this field unless you want to use a different CNI.
-<5> The name of the Linux bridge configured on the node.
+<5> The name of the Linux bridge configured on the node. The default Linux `NodeNetworkConfigurationPolicy` (NNCP) bridge name is `br1`.
```

Comment:
So then I'm confused why this bug was opened.  In the description of the bug, the reporter wrote:

`The following bridge NAD's "bridge": "bridge-interface" should be changed to "bridge": "br1", and  "bridge.network.kubevirt.io/bridge-interface" should be changed to "bridge.network.kubevirt.io/br1" The reason is that the Linux bridge NNCP setting is "name: br1".`

I understood from this that we should write `br1` because that is the actual name that appears. But technically then what we had before was fine, since the user can give their own name?

Reply:
@yossisegev  please let me know

Reply:
No, the previous version was not fine. If a user applied it, then the NAD he created would be meaningless because it was searching for a node's bridge interface called "bridge-interface", while there name of the bridge interface, as created by the NNCP, is "br1".
The purpose of this PR is to ensure consistency between the relevant resources, which would make them clearer to understand, hence make the documentation clearer.
So the user can indeed give their own names, but they would have to be consistent between the NAD and the NNCP.
So I would replace
```The default Linux `NodeNetworkConfigurationPolicy` (NNCP) bridge name is `br1`.```
with something more like
```
The name should match the name of the bridge interface, as was defined in the NodeNetworkConfiguratonPolicy (in `spec.desiredState.interfaces[0].name`)
```
I hope I managed to clarify.